ui/components/app/multichain-transaction-details-modal/multichain-transaction-details-modal.tsx

```diff
@@ -122,31 +132,44 @@ export function MultichainTransactionDetailsModal({
     ) : null;
 
   const amountComponent = (
-    {
-      amount,
-      unit,
-    }: {
-      amount: string;
-      unit: string;
-    },
-    title: string,
+    asset:
+      | {
+          amount: string;
+          unit: string;
+        }
+      | undefined,
+    label: string,
     dataTestId: string,
-  ) => (
-    <Box display={Display.Flex} justifyContent={JustifyContent.spaceBetween}>
-      <Text variant={TextVariant.bodyMd} fontWeight={FontWeight.Medium}>
-        {title}
-      </Text>
-      <Box
-        display={Display.Flex}
-        flexDirection={FlexDirection.Column}
-        alignItems={AlignItems.flexEnd}
-      >
-        <Text variant={TextVariant.bodyMd} data-testid={dataTestId}>
-          {amount} {unit}
+  ) => {
+    if (!asset) {
+      return null;
+    }
+
+    return (
+      <Box display={Display.Flex} justifyContent={JustifyContent.spaceBetween}>
+        <Text variant={TextVariant.bodyMd} fontWeight={FontWeight.Medium}>
+          {label}
         </Text>
+        <Box
+          display={Display.Flex}
+          flexDirection={FlexDirection.Column}
+          alignItems={AlignItems.flexEnd}
+        >
+          <Text variant={TextVariant.bodyMd} data-testid={dataTestId}>
+            {asset.amount} {asset.unit}
+          </Text>
+        </Box>
       </Box>
-    </Box>
-  );
+    );
+  };
+
+  const typeToTitle: Partial<Record<TransactionType, string>> = {
```

Comment:
Very nit: but given that we renamed `title` to `label` above, we could have used the same here, so `typeToLabel` 😅

Reply:
Also, what about using the same pattern that `getStatusColor` here? We would also use a `switch` and have a `default` case, for the transaction type we don't support yet. (IDK what label to use for those, but right now, if the Snap send a `bridge:send` this would probably output a `undefined`, right?)

Reply:
We are not mapping bridges at the moment.
I will get this `nit` into my next PR. I want to merge this so it makes the RC. 🙏 
Is it ok with you?